### ISSUE TITLE
About 3d Densenet load pretrain model conv filter weight data

### ISSUE THREAD

**Issue by WSW55** [rubythalib33/3D-Torchvision#2]
Dear Author,I really need your help. I am using your pretrain 3d densenet code for my classify task on 3d medical MRI image. I can't understand your pre 3d densenet  why using this way load its conv filter weight data: conv3.weight.data = torch.stack([model.conv2.weight.data], dim=2). Through this way the shape of conv3d's weight data is [1,3,3],the location 0 why not 3?Please hlep me, thank you very much !

**Comment by rubythalib33** [comment]
because load the imagenet pretrained from conv2d and I use weight data of [1,3,3] so the computation does'nt explode on low level device

**Comment by WSW55** [comment]
thank you for your reply, but, whether the third dimension information can be obtained and integrated using weight data of [1,3,3] to conv ？

**Comment by rubythalib33** [comment]
it can be
